ckanext/validation/jobs.py

```diff
@@ -180,43 +180,44 @@ def _load_dataframe(data, extension):
     return df
 
 
-def _transpose_dataframe(df):
-    # Transpose table
-    if len(df.columns) == 0:
-        transposed = df
-    else:
-        transposed = df.T
-    return transposed
-
+def _read_csv_file(data, extension=None):
+    try:
+        return pandas.read_csv(open(data, 'rb'), header=None, index_col=None)
+    except Exception as e:
+        extension = "(" + extension + ")" if extension else ""
+        raise t.ValidationError({
+            'Format': [
+                'Could not read your CSV file. Are you sure your specified '
+                'format ' + extension + ' is correct? (' + str(e) + ')'
```

Comment:
I would use python format here instead of manual string concat.
Maybe exception details should be logged istead of sending back to the user via UI?